docker_cli/_mount.py

```diff
@@ -0,0 +1,48 @@
+import os
+from typing import Union
+
+
+class Mount:
```

Comment:
Let's rename this class to `BindMount` to be consistent with Docker's nomenclature

https://docs.docker.com/storage/bind-mounts/

```suggestion
class BindMount:
```